dsl/jpql/src/test/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/expression/LengthDslTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.linecorp.kotlinjdsl.dsl.jpql.expression
+
+import com.linecorp.kotlinjdsl.dsl.jpql.queryPart
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expression
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expressions
+import org.assertj.core.api.WithAssertions
+import org.junit.jupiter.api.Test
+
+class LengthDslTest : WithAssertions {
+    private val stringExpression1: Expression<String> = Expressions.value("string1")
+
+    @Test
+    fun `length() with a string expression`() {
```

Comment:
Tests for DSL methods must check for all the possible cases of a parameter, so there must be one test for each overloaded method.